tests/adm_and_own_jobs_test.py

```diff
@@ -113,10 +116,13 @@ def _login_as_user(user):
 
     def _make_user_adm(uid):
         from sirepo.pkcli import roles
+        import sirepo.auth_role
 
         roles.add(uid, auth_role.ROLE_ADM)
         with srunit.quest_start() as qcall:
-            r = qcall.auth_db.model("UserRole").search_all_for_column("uid")
+            r = qcall.auth_db.model("UserRole").search_all_for_column(
+                "uid", role=sirepo.auth_role.ROLE_USER
```

Comment:
This should be ROLE_ADM. The test doesn't make sense the way it was written or with ROLE_USER. The check is for all ROLE_ADM rows.